packages/sw-background-sync-queue/src/lib/queue.js

```diff
@@ -16,24 +17,39 @@ class Queue {
 	}
 
 	async push(request, config){
-		let localConfig = Object.assign({}, _config, config);
-		let hash = stringHash(request.url + JSON.stringify(request.headers) + await request.text() + Date.now());
-		_queue.push(hash);
+		try{
+			let localConfig = Object.assign({}, _config, config);
+			let hash = stringHash(request.url + JSON.stringify(request.headers) + await request.text() + Date.now());
```

Comment:
`JSON.stringify(request.headers)` - are you sure this doesn't return an empty object?